### PR TITLE
RDAPI_17369: migrate Apigtw and APIMng release notes

### PR DESCRIPTION
**Pull request by andreamussap**

### CODE REVIEW

File: content/en/docs/apigtw_releasenotes/_index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+    "title": "API Gateway 7.8 Release Notes",
+    "linkTitle": "API Gateway 7.8 Release Notes",
+    "date": "2019-09-20",
```
Comment (alexearnshaw) [inline content/en/docs/apigtw_releasenotes/_index.md]: In this file and the one for api mgr, we need to add the no_list option and also can we remove the version number from the linkTitle so it doesn't appear in the left nav (we can keep it in the title).

Also probably need to add a weight option to both release notes so they appear in a better place in the nav? I'd suggest before the apache cassandra guide for now?

Comment (andreamussap) [reply content/en/docs/apigtw_releasenotes/_index.md]: RE weight option -> I'm confused with this. I looked at md files and cass_admin, apip_admin, and apip_relnotes index files have weight: 9; both APIP_docker and APIP_HA "weight":"6"!! I don't get how this work, so I've added weight: 9 to apigtw and apimng relnotes as well. Don't know where they're going to be placed in the menu.

RE no_list option -> why you want to add this? The only think they show is the 'fixed issues' page. I think it's ok to show this in the footer ??

Comment (alexearnshaw) [reply content/en/docs/apigtw_releasenotes/_index.md]: @andreamussap I think the issue is that you are using "weigh" not "weight" in the file ;) in terms of how it works, there's some more info here https://www.docsy.dev/docs/adding-content/navigation/#section-menu

On the no_list option I just think it looks a bit weird having this autogenerated listing at the bottom of a page full of content, as to me it looks like part of the last section (Support services) which it isn't meant to be. So I think best to turn it off in index pages with content, and use it only in index pages with no or little content?

**Comment by andreamussap** [comment]
Related PR - https://github.com/Axway/axway-open-docs/pull/120

**Pull request merged**